deploy/server.js

```diff
@@ -69,7 +71,59 @@ if (process.env['DATA_SOURCE'] !== 'mock') {
   });
 }
 
-app.get('*', (req, res) => {
+let clusterApiProxyOptions = {
+  target: virtMeta.clusterApi,
+  changeOrigin: true,
+  pathRewrite: {
+    '^/cluster-api/': '/',
+  },
+};
+
+let inventoryApiProxyOptions = {
+  target: virtMeta.inventoryApi,
+  changeOrigin: true,
+  pathRewrite: {
+    '^/inventory-api/': '/',
+  },
+};
+
+let inventoryPayloadApiProxyOptions = {
+  target: virtMeta.inventoryPayloadApi,
+  changeOrigin: true,
+  pathRewrite: {
+    '^/inventory-payload-api/': '/',
+  },
+};
+
+if (process.env['NODE_ENV'] === 'development') {
+  clusterApiProxyOptions = {
```

Comment:
Or I suppose you could just replace the `target` properties here since you already have a condition for dev mode.